site/src/guide.md

```diff
@@ -556,7 +599,9 @@ Regarding 3:
 
 Let's look at some examples of how this plays out, starting with the synchronous interruption case:
 
-```tut
+```tut:reset
```

Comment:
I had to reset here as tut wasn't happy with multiple implicit `ContextShift` values.